synapsemonitor/__main__.py

```diff
@@ -133,8 +135,29 @@ def build_parser():
     return parser
 
 
+def get_auth_token() -> str:
+    """Get Synapse personal access token from environmental
+    variables, if available.
+
+    Returns:
+        str: Synapse personal access token or None
+    """
+    auth_token = None
+    if os.getenv("SYNAPSE_AUTH_TOKEN") is not None:
```

Comment:
The synapse client already does this by default - So we don't need this section.

Reply:
Is it stored under synapseclient.client.{variable_name} or something like that?  Since we are not calling syn.login() without parameters, the default behavior of the login function isn't activated.  And I want to ensure that the function looks for SYNAPSE_AUTH_TOKEN first, then secrets, then the config file path

Reply:
Removed the get_auth_token function.  Now it looks for secrets first but since it's the SYNAPSE_AUTH_TOKEN anyway, I think that's fine